### PR TITLE
Update contributor guidelines for hacktober

### PR DESCRIPTION
**Pull request by ann-kilzer**
No issue.

**Proposed Changes**

With the increased traffic from hacktober, I wanted to clarify contribution instructions for newcomers.

**Testing Plan*

Preview the markdown.

### CODE REVIEW

**Comment by ann-kilzer** [comment]
We haven't heard an answer to our prior questions, but If you're not a fluent or native speaker and the document is simply run through Google Translate, I'd like to close this PR. Users can already use Google translate plugins if they wish to translate the site. We need human translation here.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,11 +6,15 @@ Welcome and thanks for your interest in contributing to our project. 🙇‍♀
 
 The tag "good first issue" and "help wanted" highlights some tickets that we'd love help with. Feel free to drop us a note if you'd like to work on these, especially if you'd like to tackle a more complex issue where it might be helpful to discuss approach. For privacy reasons, we don't allow Airtable access to the general public, but we're happy to answer questions about the table structure for contributors.
 
-Please check with us before working on tickets in the "Icebox / Ideas" column or with the "backburner" label.
+Please check with us before working on tickets in the "Icebox / Ideas" column or with the "backburner" label. Please also check before working on a ticket already assigned to somebody.
+
+## Using git
+
+We prefer merges instead of rebasing. Merges to master will be squashed. Please don't use `force-push` as it messes with Pull request comments and makes it hard for others to look at your branch.
```
Comment (tuttiq) [inline CONTRIBUTING.md:13]: Oh, I didn't know about this preference, I was using rebase myself most of the time lol I'll stick to merge then 🙇

**Review by tuttiq — APPROVED** [review]
LGTM!

File: CONTRIBUTING.md (hunk 1 above)
Comment (ann-kilzer) [reply CONTRIBUTING.md:13]: We could change it if you disagree. I've set github to auto-squash changes when merging to master via the button, and force-push created a lot of headaches at my last job so I really encourage people to find a different way.

**Pull request merged**